src/content/en/2021/caching.md

```diff
@@ -17,4 +19,140 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Over the past decade ...
+
+
+## CDN Cache adoption
+
+Talk about CDN adoptions, 14% of desktop pages use a Cloudflare CDN.
+
+{{ figure_markup(
+  image="to-do.png",
+  caption="Top CDNs adoption",
+  description="TODO",
+  chart_url="TODO",
+  sheets_gid="58739923",
+  sql_file="todo.sql"
+) }}
+
+## Service Worker adoption
+
+Talk about Service Worker adoptions, 1.2% of desktop pages registered a Service Worker, but 9% of mobile pages in the top 1000 registered a SW.
+
+{{ figure_markup(
+  image="to-do.png",
+  caption="Service Worker adoption,
+  description="TODO",
+  chart_url="TODO",
+  sheets_gid="802609299",
```

Comment:
@rviscomi sheet `802609299 ` would be possible have a figure showing the Service Worker adoption.